gettsim/benefits/arbeitsl_geld_2.py

```diff
@@ -9,8 +15,14 @@ def alg2(household, params):
         parents. Income and wealth is tested for, the transfer withdrawal rate is
         non-constant.
     """
+    if params["jahr"] < 2005:
+        # warnings.warn("Arbeitslosengeld 2 existiert erst ab 2005 und wurde deshalb "
```

Comment:
Why is the warning commented out if we do not calculate anything here? Should even be an error, I think. At least once we are in the DAG world, the resulting column should never show up.

Reply:
The layout right now is: If reform does not exist, return nan, s.t. you can't keep calculating with outputs. I agree, that this is a bad layout, but gettsim does not provide the possibility to select different parts so far. So I would suggest implementing the whole error structure as soon as we have DAG in place! Till then you can run whole gettsim for example in 1995 and you get nans for all not implemented parts. You still could get taxes tariff, soli etc....

Reply:
Sounds good, no worries at this point.